src/dialogs/components/dialog/dialogField.html

```diff
@@ -0,0 +1,133 @@
+                    ng-disabled="vm.dialogField.read_only || vm.inputDisabled"
+                    theme="bootstrap">
+          <ui-select-match
+            placeholder="{{vm.dialogField.description}}">{{$select.selected[1] }}</ui-select-match>
+          <ui-select-choices
+            repeat="fieldValue[0] as fieldValue in vm.dialogField.values | filter: $select.search">
+            <span>{{fieldValue[1]}}</span>
+          </ui-select-choices>
+        </ui-select>
+      </span>
+
+      <select ng-if="vm.dialogField.options.force_single_value"
+              ng-switch-when="DialogFieldTagControl"
+              ng-model="vm.dialogField.default_value"
+              ng-disabled="vm.dialogField.read_only || vm.inputDisabled"
+              ng-change="vm.changesHappened()"
+              class="form-control"
+              ng-options="fieldValue.id as fieldValue.description for fieldValue in vm.dialogField.values">
+      </select>
+
+      <!-- Somewhat of a hack, but open angular issue using ng-att-multiple, so this is the workaround -->
+      <select ng-if="!vm.dialogField.options.force_single_value"
+              multiple
+              ng-switch-when="DialogFieldTagControl"
+              ng-model="vm.dialogField.default_value"
+              ng-disabled="vm.dialogField.read_only || vm.inputDisabled"
+              ng-change="vm.changesHappened()"
+              class="form-control"
+              ng-options="fieldValue.id as fieldValue.description for fieldValue in vm.dialogField.values">
+      </select>
+
+      <span
+        ng-if="vm.dialogField.read_only || vm.inputDisabled"
+        ng-switch-when="DialogFieldRadioButton"
+        class="btn-group">
+        <label>{{ vm.parsedOptions[vm.dialogField.name] }}</label>
+      </span>
+      <span ng-if="vm.dialogField.read_only === false || vm.inputDisabled === false"
+            ng-switch-when="DialogFieldRadioButton"
+            class="btn-group">
+        <label
+          class="btn btn-primary"
+          ng-repeat="fieldValue in vm.dialogField.values">
+          <input
+            type="radio"
+            ng-model="vm.dialogField.default_value"
+            ng-change="vm.changesHappened()"
+            ng-disabled="vm.dialogField.read_only || vm.inputDisabled"
+            name="{{ vm.dialogField.name }}"
+            value="{{ ::fieldValue[0] }}"/>
+          {{ ::fieldValue[1] }}
+        </label>
+      </span>
+
+      <p ng-switch-when="DialogFieldDateControl" class="input-group">
+        <input uib-datepicker-popup type="text" class="form-control"
+                ng-model="vm.dialogField.default_value"
+                ng-change="vm.changesHappened()"
+                is-open="open"
+                datepicker-options="vm.dateOptions" close-text="Close"/>
+        <span class="input-group-btn">
+          <button type="button" class="btn btn-default" ng-click="open = !open">
+            <i class="fa fa-calendar"></i></button>
```

Comment:
And this.

There's nothing particulalry wrong with multiple elements on one line, but if you start an element on a line by itself, close it on a line by itself as well.